tests/integration/blockchain/test_blockchain.py

```diff
@@ -1149,7 +1149,7 @@ async def test_simple_support_trending(self):
         claim1 = await self.get_claim(await self.create_claim(name="one"))
         claim2 = await self.get_claim(await self.create_claim(name="two"))
         await self.generate(1)
-        results = await self.db.search_claims(order_by=["trend_group", "trend_mixed"])
+        results = await self.db.search_claims(order_by=["trending_group", "trending_mixed"])  # test compat layer
```

Comment:
maybe outside the scope of this PR, but i would think the compat layer should be somewhere at the API level, not deep down in the db. don't have to fix it for this PR, just an observation for another PR.

Reply:
yes, agree. there are more compat changes I will organize better in another pr